app/Dao/UserDao.php

```diff
@@ -30,22 +25,11 @@ public function getUsers(): object
      */
     public function createUser(array $data): void
     {
-        $imgName = '';
-        if (isset($data['image']) && $data['image']->isValid()) {
-            $image = $data['image'];
-            $imgName = uniqid() . '_' . $image->getClientOriginalName();
-            $image->storeAs('public/images', $imgName);
-        }
-
-        $pass = $data['password'];
-        $encrypted = openssl_encrypt($pass, 'aes-256-cbc', $this->key, OPENSSL_RAW_DATA, $this->iv);
-        $encryptedData = base64_encode($encrypted);
-
         User::create([
             'name' => $data['name'],
             'email' => $data['email'],
-            'password' => $encryptedData,
-            'img' => $imgName,
+            'password' => Hash::make($data['password']),
+            'img' => $data['image']?:null,
```

Comment:
Add space before and after ternary operator. 
```suggestion
            'img' => $data['image'] ?: null,
```